src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/AutoInsert/OnAutoInsertEndpoint.cs

```diff
@@ -163,11 +163,11 @@ public RegistrationExtensionResult GetRegistration(VSInternalClientCapabilities
             TextEdit[] mappedEdits;
             if (delegatedResponse.TextEditFormat == InsertTextFormat.Snippet)
             {
-                mappedEdits = await razorFormattingService.FormatSnippetAsync(documentContext.Identifier.Uri, documentContext.Snapshot, projection.LanguageKind, edits, originalRequest.Options, cancellationToken).ConfigureAwait(false);
+                mappedEdits = await razorFormattingService.FormatSnippetAsync(documentContext, projection.LanguageKind, edits, originalRequest.Options, cancellationToken).ConfigureAwait(false);
```

Comment:
Rather than pass the HostDocumentVersion too lets stop tearing parts off the DocumentContext and just pass the whole darn thing.